polyaxon_schemas/build.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, image, build_steps=None, env_vars=None, nocache=None, git=Non
 
     @property
     def image_tag(self):
-        tagged_image = self.image.split(':')
+        tagged_image = self.image.rsplit(':', 1)
```

Comment:
I think we need to keep the same split and change the condition, basically, we need to check the following cases:

 * `len == 1` <==> latest
 * `len == 2` <==> if `tagged_image[1]` contains a `/` this means that's a port, because it should be the following case, assuming this image: `registry.foobar.com:4567/my/docker/image`, `['registry.foobar.com', '4567/my/docker/image']`, and so it should return a latest as well. Otherwise if no `/` was found, `tagged_image[-1]` should be a tag.
 * `len == 3` <==> `tagged_image[-1]` is the correct tag.

Reply:
This makes sense to me!